pkg/apis/autoheal/types.go

```diff
@@ -84,6 +84,10 @@ type AWXJobAction struct {
 	// ExtraVars are the extra variables that will be passed to job.
 	// +optional
 	ExtraVars JsonDoc
+
+	// Limit in the execution limit that will be passed to job.
```

Comment:
I don't quite understand this comment, sorry :-) , can you reword it?

Reply:
**Limit**: A host pattern to further constrain the list of hosts managed or affected by the playbook. Multiple patterns can be separated by colons (`:`). As with core Ansible, `a:b` means `in group a or b`, `a:b:&c` means `in a or b but must be in c`, and `a:!b` means `in a, and definitely not in b`.

http://docs.ansible.com/ansible-tower/latest/html/userguide/job_templates.html